Page/PageWrapper.vue

```diff
@@ -71,25 +74,34 @@ onMounted(() => {
         <slot name="title-right" />
       </template>
     </Component>
-    <slot v-if="!loading" />
 
-    <slot
-      v-else
-      name="loading"
+    <!-- Content -->
+    <ScrollArea
+      class="page-wrapper__content"
+      p="s-2! md:s-5! e-2 md:e-0"
```

Comment:
using `padding-inline-start` and `padding-inline-end` is a bit weird here... but okay :)

Reply:
It is, but PerfectScrollbar uses it, and I couldnt override it with regular `padding`, so I adapted.